index.js

```diff
@@ -889,10 +889,8 @@ let createContainer = ({resolvers = [], dependencies = {}, factories, definition
 
         if (!definition.reuse) {
             destroyObject(instance, options);
-            definition.instance = null;
-        } else {
-            delete definition.instance;
         }
+        definition.instance = null;
```

Comment:
For instance reuse we really need delete here, because definitions linked by __proto__ link, and when we write null to child, it write to child. When we delete instance from child, we still can see instance from parent